### ISSUE TITLE
Android: Crash on invoking disconnectOperation

### ISSUE THREAD

**Issue by ganeshkarthick4** [MailCore/mailcore2#1478]
Hi,
When I try to disconnect an Imapsession in Android using 
Imapsession.disconnectOperation(), my app is crashing.

_07-13 23:17:30.891: W/google-breakpad(7751): ### ### ### ### ### ### ### ### ### ### ### ### ###
07-13 23:17:30.891: W/google-breakpad(7751): Chrome build fingerprint:
07-13 23:17:30.891: W/google-breakpad(7751): 0.5
07-13 23:17:30.891: W/google-breakpad(7751): 130
07-13 23:17:30.891: W/google-breakpad(7751): ### ### ### ### ### ### ### ### ### ### ### ### ###
07-13 23:17:30.891: A/libc(7751): Fatal signal 6 (SIGABRT), code -6 in tid 9064 (IMAPService[IMA)_

Also is there an auto disconnect of session?

**Comment by dinhviethoa** [comment]
Yep. Sessions are auto disconnecting with a 30 sec or 1 minute delay (not sure of the actual value).
Are you able to build source of mailcore2 to be able to investigate the issue?
